### PR TITLE
Add support to specify host port

### PR DESCRIPTION
**Pull request by machacekondra**

### CODE REVIEW

**Comment by machacekondra** [comment]
cc @mwperina

**Comment by machacekondra** [comment]
Hello @pcuzner @jmolmo can you please review/merge if all is OK? Thank you!

**Comment by jmolmo** [comment]
LGTM

**Comment by machacekondra** [comment]
Seems reasonable, I will change, thanks.

**Comment by machacekondra** [comment]
@pcuzner done

File: runner_service/controllers/hosts.py [hunk 1]
```diff
@@ -159,16 +176,19 @@ def post(self, host_name, group_name):
...
 
         args = request.args.to_dict()
+
         if args:
             logger.debug("additional args received")
             if all(p in valid_parms for p in args.keys()):
                 if 'others' in args:
                     group_list.extend(args['others'].split(','))
+                ssh_port = args.get('port', None)
             else:
```
Comment (pcuzner) [inline runner_service/controllers/hosts.py]: Sorry - I didn't notice this before, but can you validate the port before we pass it on to use it in the inventory - i.e. must be numeric

Comment (machacekondra) [reply runner_service/controllers/hosts.py]: Done.

**Pull request merged**